cirq-core/cirq/devices/thermal_noise_model.py

```diff
@@ -160,7 +161,8 @@ def _validate_rates(qubits: set[cirq.Qid], rates: dict[cirq.Qid, np.ndarray]) ->
             )
 
 
-@dataclasses.dataclass
+@dataclasses.dataclass(eq=False)
```

Comment:
I can't see any dataclass specification in the class code.  Please delete the decorator.

Reply:
Done in 0a1332d780a300be4395f2120242f4ffc36f1d0e.